src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1591,8 +2019,10 @@ subroutine gsi_fv3ncdf_read(grd_ionouv,cstate_nouv,filenamein,fv3filenamegin)
           deallocate(uu2d_layout)
         enddo
       else
+       !write(6,*)' gsi_fv3ncdf_read_wang2: ',ilevtot,varname
```

Comment:
Remove these commented out print statements with `wangN` in the print, assuming they were for debugging